tuneme/templates/core/article_page.html

```diff
@@ -63,6 +63,17 @@
 </div>
 </div>
 
+{% has_questions self as questions %}
+{% if questions %}
+<div class="block {{self.get_effective_extra_style_hints|default:"orange"}}">
```

Comment:
In an article level you want to get the colour from it's parent so you should do `self.get_parent_section.get_effective_extra_style_hints`